src/main/java/org/prebid/server/auction/ImplicitParametersExtractor.java

```diff
@@ -83,6 +95,17 @@ public String ipFrom(HttpServerRequest request) {
         return ip;
     }
 
+    /**
+     * Check if given IP address is a private IP.
+     */
+    private static boolean checkPrivateIp(String ip) {
```

Comment:
Minor. Pls rename method name to respect a predicate action.
It is not clear what the result of `checkXXX` method should be.